examples/src/main/scala/org/apache/spark/examples/DFSReadWriteTest.scala

```diff
@@ -46,9 +47,7 @@ object DFSReadWriteTest {
   private val NPARAMS = 2
 
   private def readFile(filename: String): List[String] = {
-    val lineIter: Iterator[String] = fromFile(filename).getLines()
-    val lineList: List[String] = lineIter.toList
-    lineList
+    Utils.tryWithResource(fromFile(filename))(_.getLines()).toList
```

Comment:
I think you should call `getLines().toList` right? because `getLines()` is lazy and it won't consume before you call `toList`.

Reply:
done ~